db/seeds.rb

```diff
@@ -321,18 +321,33 @@ def setup_fake_health_data
 def setup_hmis_admin_access
   return unless ENV['HMIS_HOSTNAME'].present?
 
-  hmis_ds = GrdaWarehouse::DataSource.source.where.not(hmis: nil).first_or_create do |ds|
-    ds.hmis = ENV['HMIS_HOSTNAME']
+  # Create HMIS Administrator role
+  hmis_admin_role = Hmis::Role.where(can_administer_hmis: true).first_or_create do |role|
```

Comment:
This is a more general issue in this seeds script but I wonder if it would be best to always use the bang-variant (`first_or_create!`) to throw an exception if the save fails.